docs/landing/concepts/plugins.md

```diff
@@ -286,3 +286,123 @@ so now we just need to add our new creations to our plugin's `config.json` file:
 so now, visiting `/our-custom-route/magic-id` you should see:
 
 ![custom-plugin-pages-and-layouts](./../assets/plugins-custom-layouts-and-pages.png)
+
+## Checkout plugins validation (payment, shipping) <Badge text="from v0.9" type="info"/>
+
+When you're developing a payment or shipping plugin and you need custom validation - we got you!
```

Comment:
```suggestion
When you're developing a payment or shipping plugin and you need custom validation - we got you!

Since the very cool validation library called [vuelidate](https://vuelidate.js.org/) is used, our life is simpler. 
Several built-in validaton rules are at hand, and writing a custom rule should not be a big deal as well (see how to do it [here](https://vuelidate-next.netlify.app/custom_validators.html)).
```